vizro-ai/docs/pages/user-guides/dashboard-iterate-and-deploy.md

```diff
@@ -0,0 +1,111 @@
+# Refine a generated dashboard
```

Comment:
@nadijagraca Let's drop this whole page for now. Can add it back once the "pydantic to python" PR is merged.